apps/spruce/.env-cmdrc.json

```diff
@@ -38,21 +39,27 @@
   },
   "devLocal": {
     "NODE_ENV": "development",
+    "REACT_APP_REMOTE_ENV": "false",
     "REACT_APP_PARSLEY_URL": "http://localhost:5173",
     "REACT_APP_SPRUCE_URL": "http://localhost:3000",
     "REACT_APP_UI_URL": "http://localhost:9090"
   },
   "devStaging": {
     "NODE_ENV": "development",
+    "REACT_APP_REMOTE_ENV": "true",
     "REACT_APP_PARSLEY_URL": "http://parsley-staging.corp.mongodb.com",
-    "REACT_APP_SPRUCE_URL": "http://localhost:3000",
-    "REACT_APP_UI_URL": "https://evergreen-staging.corp.mongodb.com"
+    "REACT_APP_SPRUCE_URL": "https://spruce-local.corp.mongodb.com",
+    "REACT_APP_UI_URL": "https://evergreen.staging.corp.mongodb.com",
+    "REACT_APP_HONEYCOMB_BASE_URL": "https://ui.honeycomb.io/mongodb-4b/environments/staging",
+    "REACT_APP_CORP_LOGIN_URL": "https://login.corp.mongodb.com/login"
   },
   "devProduction": {
     "NODE_ENV": "development",
+    "REACT_APP_REMOTE_ENV": "true",
```

Comment:
do we want to wait on this until we have a new corpsecured prod URL?

Reply:
This is still fine since 1. It technically still is a remote env and it works the same way either way.